shell/common/node_bindings.cc

```diff
@@ -122,6 +123,58 @@ void stop_and_close_uv_loop(uv_loop_t* loop) {
 
 bool g_is_initialized = false;
 
+bool IsPackagedApp() {
+  base::FilePath exe_path;
+  base::PathService::Get(base::FILE_EXE, &exe_path);
+  base::FilePath::StringType path = base::ToLowerASCII(exe_path.value());
+
+#if defined(OS_WIN)
+  return path == FILE_PATH_LITERAL("electron.exe");
+#else
+  return path == FILE_PATH_LITERAL("electron");
```

Comment:
Its still not sufficient, helper executables end with `Electron Helper` etc, in this case `Electron Helper (Renderer)` comparing to `electron` wouldn't be right. The tests we have right now will pass even for this condition because we pass unilaterally invalid arguments in the test case. This issue will show up when the app is packaged.

Reply:
Should be `!=` here, the change seems to be reverted back when resetting.

Reply:
@zcbenz if i make if the other way around (the way you're suggesting), both specs fail, meaning it's returning packaged when it shouldn't be 🤔